CollectedClientResponse.csv

```diff
@@ -0,0 +1,2 @@
+"Recipient","Type_of_confirmation","Action","Response_date_time","Feedback","Expire_date_time","Process_name","Step_name","Client_name"
+"ivan.sadovoy@solvexia.com","Acknowledgement","Acknowledge","22 Feb 2018 15:18:25","","23 Feb 2018 15:09:05","Email confirmation test","1.2 Email Confirmation","svx.ivan.sadovoy"

```

Comment:
We should create or use a separate mail box or alias dedicated for api tests instead of using individual email address.

Reply:
done